rest_framework_json_api/renderers.py

```diff
@@ -433,6 +427,13 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
 
         serializer = getattr(serializer_data, 'serializer', None)
 
+        # Build a list of included resources
+        included_resources = utils.get_default_included_resources_from_serializer(serializer)
+        include_resources_param = request.query_params.get('include') if request else None
+        if include_resources_param:
+            extra = filter(lambda r: r not in included_resources, include_resources_param.split(','))
```

Comment:
I don't believe this will result in behavior compliant with the spec. The spec states that "If an endpoint supports the include parameter and a client supplies it, the server MUST NOT include unrequested resource objects in the included section". So we should build `included_resources` from `JSONAPIMeta.included_resources` for a default but if `include` exists in the url then its value should be used instead.